packages/messenger-widget/src/hooks/messages/acknowledge/AcklowledgementManager.ts

```diff
@@ -0,0 +1,90 @@
+import { Account, ProfileKeys } from '@dm3-org/dm3-lib-profile';
+import { MessageModel } from '../useMessage';
+import {
+    createReadOpenMessage,
+    createReadReceiveMessage,
+} from '@dm3-org/dm3-lib-messaging';
+import { ContactPreview } from '../../../interfaces/utils';
+
+//Class that takes care of acknowledging messages
+export class AcknowledgmentManager {
```

Comment:
```suggestion
export class AcknowledgementManager {
```

Also, please rename file: Ack**l**owledgementManager.ts -> Ack**n**owledgementManager.ts (replace L with N)